tests/src/test/java/org/eclipse/hono/tests/IntegrationTestSupport.java

```diff
@@ -73,7 +73,7 @@
     /**
      * The default number of milliseconds to wait for a response to an AMQP 1.0 performative.
      */
-    public static final int DEFAULT_AMQP_TIMEOUT = 400;
+    public static final int DEFAULT_AMQP_TIMEOUT = 1000;
```

Comment:
please revert this change, we are already applying a factor > 1 to this value when running on test environment